core/templates/pages/topic-editor-page/editor-tab/topic-editor-tab.directive.ts

```diff
@@ -152,7 +173,23 @@ angular.module('oppia').directive('topicEditorTab', [
             }
           };
 
+          $scope.togglePreview = function() {
+            $scope.showPreview = !($scope.showPreview);
```

Comment:
$scope.previewIsShown perhaps.

Also maybe include, in the name, what exactly is being previewed here -- I can't really tell.

Reply:
Made this more clear. Done.